sematic/types/types/list.py

```diff
@@ -18,6 +17,8 @@
     value_to_json_encodable,
 )
 
+MAX_SUMMARY_BYTES = 2**17  # 128 kB
```

Comment:
Given that the display just shows all the elements stacked on top of one another for things like dataclasses, the page gets really long if you have too many. Things still load and render really fast, it just makes the page unreasonably large.